templates/common/on-prem/files/NetworkManager-resolv-prepender.yaml

```diff
@@ -20,13 +20,49 @@ contents:
     {{end -}}
     {{end -}}
 
+    function retry {
+        attempts=5
```

Comment:
I see no reason not to retry forever.  That's what we do in other places.

Reply:
That's a fair suggestion. While I can't easily see what could take so long so that 5 attempts are not enough but "forever" is, I agree that we already have a path for retrying in our codebase that uses a simple `while !` so I just changed to use the same